test/unit_Polygon.jl

```diff
@@ -391,6 +391,10 @@ for N in [Float64, Float32, Rational{Int}]
     m = N[4 0; 6 2; 4 4]'
     P = VPolygon(m)
     @test is_cyclic_permutation(vertices_list(P), [N[4, 0], N[6, 2], N[4, 4]])
+
+    # test concrete projection
+    V = VPolygon([N[0, 1], N[1, 0], N[-1, 0]])
+    @test project(V, [1]) == VPolygon([N[-1], N[1]])
```

Comment:
Can you add a test for `V = VPolygon([N[1, 0], N[1, 1]]); project(V, [1])`? This shows that in general you can get duplicates. I wonder if we should add an option to apply a convex hull.

Reply:
> I wonder if we should add an option to apply a convex hull.

note that the VPolygon constructor has such option but the VPolytope does not (such option would require loading Polyhedra).